### ISSUE TITLE
Support for "JCGM" prefix for JCGM documents

### ISSUE THREAD

**Issue by ronaldtse** [relaton/relaton-bipm#57]
```sh
$ bundle exec relaton fetch "JCGM 101:2008"
[relaton] `JCGM 101:2008` does not have a recognised prefix
No matching bibliographic entry found
```

```sh
$ bundle exec relaton fetch "BIPM JCGM 101:2008"
[relaton-bipm] (BIPM JCGM 101:2008) Fetching from Relaton repository ...
[relaton-bipm] WARNING: Incorrect reference: `JCGM 101:2008`
Expected one of [OUTCOME, BROCHURE, METROLOGIA] at line 1 char 1.
```

Looks like we have to add pubid support for BIPM / JCGM documents?

**Comment by andrew2net** [comment]
@ronaldtse I'll add the `JCGM` prefix but:
- [format](https://github.com/relaton/relaton-bipm#reference-structures-for-cctf-ccds-cgpm-cipm-documents) for meeting references should be like `JCGM Meeting 101 (2008)`
- There is no 101 meeting in the [bipm-data-outcome](https://github.com/metanorma/bipm-data-outcomes/tree/main/jcgm/meetings-en) dataset

**Comment by andrew2net** [comment]
Fixed in v1.16.5
```sh
$ relaton fetch --no-cache 'JCGM Meeting 11 (2006)'
[relaton-bipm] (JCGM Meeting 11 (2006)) Fetching from Relaton repository ...
[relaton-bipm] (JCGM Meeting 11 (2006)) Found: `JCGM 11st Meeting (2006)`
<bibdata type="proceedings" schema-version="v1.2.4">
  ...
```
